cli/pkg/earthfile.go

```diff
@@ -14,7 +14,7 @@ type Earthfile struct {
 	Path           string               `json:"path,omitempty"`
 	SourceLocation *spec.SourceLocation `json:"sourceLocation,omitempty"`
 	Targets        []spec.Target        `json:"targets,omitempty"`
-	UserCommands   []spec.UserCommand   `json:"userCommands,omitempty"`
+	Functions      []spec.Function      `json:"functions,omitempty"`
```

Comment:
Seems that we are not using this field maybe it makes sense to remove it ?
cc @jmgilman